OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs

```diff
@@ -745,5 +750,68 @@ void MakeMouseFocusSettingsLive()
 			else
 				Game.Renderer.ReleaseWindowMouseFocus();
 		}
+
+		void InitHotkeyRemapDialog(Widget panel)
+		{
+			var label = new CachedTransform<HotkeyDefinition, string>(hd => hd.Description + ":");
+			panel.Get<LabelWidget>("HOTKEY_LABEL").GetText = () => label.Update(selectedHotkeyDefinition);
+
+			var duplicateNotice = panel.Get<LabelWidget>("DUPLICATE_NOTICE");
+			duplicateNotice.TextColor = ChromeMetrics.Get<Color>("NoticeErrorColor");
+			duplicateNotice.IsVisible = () => !isHotkeyValid;
+			var duplicateNoticeText = new CachedTransform<HotkeyDefinition, string>(hd => (hd != null) ? duplicateNotice.Text.F(hd.Description) : duplicateNotice.Text);
```

Comment:
Nit: redundant () around the hd != null check

Reply:
Fixed.